server/src/config.js

```diff
@@ -0,0 +1,45 @@
+const configData = {
```

Comment:
Can we remove the config file, or blank out the secret keys. @matt-auckland and I will do a PR that pulls them out into a seperate file to the regular config shortly.